app/src/main/java/org/wikipedia/analytics/eventplatform/UserContributionEvent.kt

```diff
@@ -0,0 +1,96 @@
+package org.wikipedia.analytics.eventplatform
+
+import com.google.gson.annotations.SerializedName
+
+class UserContributionEvent : Event(SCHEMA_NAME, STREAM_NAME) {
+    @SerializedName("action") private var action: String = ""
```

Comment:
Is it really necessary to add `@SerializedName("action")` if the name is already `action`?